process/incident_management.md

```diff
@@ -0,0 +1,273 @@
+
+The Incident Commander can elect someone to be the "scribe" during the incident. The duty of the scribe is to report in the RCA document all the important information (technical details, timeline, ...) while the Incident Commander is focused on helping with the coordination.
+
+#### Responsibilities
+The coordination work under the responsibility of the Incident Commander is as follows:
+1. If the incident has been uncovered by SRE and doesn't have an existing "external" ticket (Jira or ICM for ARO) then the Incident Commander will create a Jira ticket to track the overall life of the incident.
+2. Right after the Incident Commander will create a dedicated Slack channel:
+  * Making sure that we have a dedicated IM channel opened specifically for the ongoing incident
+    1. Channel is public
+    2. Naming convention for the IM channel should start with the ticket number and the clusterid if available.
+       * Examples:
+         * #inc1096029-<CLUSTERID>
+         * #OHSS-1234-title
+    3. The description of the IM channel should record:
+      * Incident Commander
+      * Incident Tech Lead
+      * Bridge link
+      * A small description of the issue
+    4. Making sure the right people are invited in the slack channel (including customer support if need be)
+    5. Making sure that all communications and information flows are recorded in the IM channel (it will help with the Post Mortem Review process and making sure that we have the right timeline)
+  3. Using IM, post a message to the relevant channel with:
+    * **link to bridge**
+    * **link to the dedicated slack channel (see point 2.)**
+    * **short description of issue (which cluster, what’s failing)**
+  4. Post an incident notification to the customer(s)
+    * NOTE: If the ServiceLog facility is unavailable, work with the CEE Lead to open a proactive Support Case. If the incident may affect a large number of clusters / customers, the CEE Lead should escalate through their management chain to post a Customer Portal banner announcement.
+  5. Create right away the RCA document and ensure that it contains the relevant information which would record the high level timeline of events and help during handovers if the incident becomes a long running one.
+  6. Designate the Incident Owner / confirm the First Responder will be that, announce it officially in the different channels, document it in the RCA document and assign the Incident Jira ticket to the Incident Owner.
+  7. Designate an Incident Technical Lead (on the first shift it will be the Incident Owner but then if the incident spans across multiple shifts this role will be handed over to the next)
+  8. Publicly announce who is Incident Commander (IC) in the dedicated slack channel.
```

Comment:
remove ref of Slack and just mention IM channel

Reply:
updated in [f799548](https://github.com/operate-first/sre/pull/17/commits/f79954870cd4360b9ede03ca11adbd53bd87232d)